p751toolbox/curve.go

```diff
@@ -452,6 +536,38 @@ func (xR *ProjectivePoint) ThreePointLadder(curve *ProjectiveCurveParameters, xP
 	return xR
 }
 
+/**
+Update: This is the right-to-left method for computing the x-coordinate of P+[k]Q.
```

Comment:
Update is not a valid comment IMO. I would modify the existing description.

Reply:
Agree with that.